docs/extras/nandrestore.md

```diff
@@ -1,36 +1,39 @@
-## Restore a nand backup to the switch
+## Restore a NAND backup to the switch
 
-!!! danger "Warning:"	
-	-This will reset all of your saves, games, system version and other system settings back to a point when you made the the nand backup. Keep this in mind, as you probably don't have to restore a nand backup unless you have bricked your switch or want to go back online safely after using cfw.
+!!! Danger "Warning:" 	
+	-This will reset your system version, all of your saves, games, and other system settings back to a point from when you made the the NAND backup. Keep this in mind, as you probably don't have to restore a NAND backup unless you have bricked your switch or want to go back online safely after using cfw.
 	
-	-If you're going to restore an old nand which will downgrade your firmware it's best to create a second NAND backup before restoring the first one in case something goes wrong.
+	-If you're going to restore an old NAND which will downgrade your firmware it's best to create a second NAND backup before restoring the first one in case something goes wrong.
 
 ### What you need:
-- Your `Rawnand.bin` (Combined or in 15 or 30 parts)
+- Your `RawNAND.bin` (Combined or in 15 or 30 parts)
```

Comment:
Should be `rawnand.bin`. This is a repeating issue, so I'm only adding this once, but fix this across the board please.

Reply:
fixed both cases of it